docs/source/ipex/inference.mdx

```diff
@@ -0,0 +1,46 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Inference
+
+Optimum Intel can be used to load models from the [Hub](https://huggingface.co/models) and create pipelines to run inference with IPEX optimization (including patching, weight prepack and graph mode) on a variety of Intel processors (currently only support for CPU)
```

Comment:
What's the plan for the model in the next release ? From my understanding we will stop using TorchScript, I'd prefer to wait for the next release once we have something that won't get deprecated, before adding it to the documentation

Reply:
I don't think waiting for the next release is the best option since the `torch.compile` is not under our control. Currently, `torch.compile` is not working on all models for all tasks; many performance regressions need to be fixed. We will apply the `torch.compile` one by one for models. Changing all models to compile without any issues is impossible for now, so it's a long-term project.